Tools/autotest/arducopter.py

```diff
@@ -3662,19 +3662,19 @@ def Parachute(self):
                      0, # param6
                      0 # param7
                      )
-        self.wait_statustext('BANG! Parachute deployed', timeout=2)
+        self.wait_statustext('BANG', timeout=60)
```

Comment:
I changed `BANG! Parachute deployed` to `BANG` because the status text emitted by vehicle is actually _BANG!  Parachute deployed_ (having two spaced instead of one after the exclamation mark). Of course a better fix would be to fix the later at vehicle end which I will be doing in another PR. This change gets the things working for now.

The second change was because the test was failing at my machine and when I went through the logs I saw the status text being emitted by vehicle but was not captured by the test. I suspected it was happening because of little delay between the time when the text was emitted and the time the test captures it. Hence, I increased the time. I agree 60 seconds is too large. Will try with something lesser.